src/js/App/Sidenav/NavigationItem.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import NavigationItem  from './NavigationItem';
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+
+describe('NavigationItem', () => {
+    it('should render coorectly', () =>{
```

Comment:
Can you also add tests for
* `parrent` with value and `itemID` with value
* `parrent` with value `itemID` without value
* `parent` withou value `itemID` with value
* `navigate` value set

Reply:
done